pallets/msa/src/lib.rs

```diff
@@ -824,6 +824,14 @@ where
 				)?;
 				return ValidTransaction::with_tag_prefix(TAG_PREFIX).and_provides(who).build()
 			},
+			Some(Call::revoke_msa_key { key, .. }) => {
+				const TAG_PREFIX: &str = "KeyRevocation";
```

Comment:
Minor comment but I think it might be a good idea to move the body into to a function like to reduce duplication. Something like:

```
handle_call_priority(who, TAG_PREFIX)
```